R/pagination.R

```diff
@@ -1161,31 +1279,30 @@ paginate_to_mpfs <- function(obj,
   ## but we will if we ever allow force_paginate to do horiz
   ## pagination.
   if (length(fpags) > 1) {
-    # Correction for case we are entering here (page_by?)
-    if (!is.null(page_num)) {
-      prov_footer(obj) <- head(prov_footer(obj), -1)
-      fpags <- do_forced_paginate(obj)
-    }
-    deep_pag <- paginate_to_mpfs( # what about the other parameters?
-      fpags,
+    deep_pag <- lapply(
+      fpags, paginate_to_mpfs,
       pg_size_spec = pg_size_spec,
       colwidths = colwidths,
       min_siblings = min_siblings,
       nosplitin = nosplitin,
+      fontspec = fontspec,
       verbose = verbose,
+      rep_cols = rep_cols,
       page_num = page_num
     )
-    return(deep_pag)
+    return(unlist(deep_pag, recursive = FALSE))
```

Comment:
This needs to be reverted. Fix from yesterday, we can change the prov_footer as you suggested in the other PR (so no rerunning of do_forced_pagination

Reply:
please look at what is in the branch now (when you're back at your desk) it passes the reinstated test

Reply:
Yes but counts were problematic for other cases. I reinstated the current fix. Will merge the behaviors in the future ;)